tools/autograd/derivatives.yaml

```diff
@@ -2984,3 +2984,7 @@
 
 - name: special_spherical_bessel_j0(Tensor x) -> Tensor
   x: non_differentiable
+
+- name: reshape_copy(Tensor self, SymInt[] size) -> Tensor
```

Comment:
Would it work to make it a `copy: bool` kwarg like we did for `.to()` https://pytorch.org/docs/master/generated/torch.Tensor.to.html?highlight=#torch.Tensor.to ?

Reply:
Even if I did add the kwarg, I would still have to add an extra operator, similar to how we have `_to_copy`